Formula/mono.rb

```diff
@@ -103,6 +104,13 @@ def install
     # run directly, so we move them out of bin
     libexec.install bin/"mono-gdb.py", bin/"mono-sgen-gdb.py"
 
+    # TODO: Try updating certs. `mozroots` is deprecated so should try to fix `cert-sync`
+    # but need to patch directory. Probably want to find way to clear certs before bottling.
+    #
+    # FIXME: cert-sync tries to write to /usr/ path. Not sure about `mozroots`.
+    # system bin/"cert-sync", Formula["ca-certificates"].pkgetc/"cert.pem"
+    system bin/"mozroots", "--import", "--sync" if OS.linux?
```

Comment:
`mozroots` does default to user directory (`~/.config/.mono/certs`) while `cert-sync` defaults to system directory (`/usr/share/.mono/certs`) so it may be able to write certificates in a temp location.

If it does help, will probably try `cert-sync --user` instead.

However, still should try to fix system location and maybe add `cert-sync` to post-install. There was an attempt to change location on macOS as default is blocked by SIP (https://github.com/mono/mono/pull/2363) but that was reverted and it wouldn't help us. May just need to manually patch.

Reply:
Still having trouble loading index:
```
Restoring package named Microsoft.NET.Sdk version='3.1.400-preview.20365.20'
  /tmp/mono--msbuild-20221004-7860-1b2jkx2/Directory.Build.props(3,3): warning : Unable to load the service index for source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json.
  /tmp/mono--msbuild-20221004-7860-1b2jkx2/Directory.Build.props(3,3): warning : Unable to load the service index for source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json.
  /tmp/mono--msbuild-20221004-7860-1b2jkx2/Directory.Build.props(3,3): warning : Unable to load the service index for source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json.
  /tmp/mono--msbuild-20221004-7860-1b2jkx2/Directory.Build.props(3,3): warning : Unable to load the service index for source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-symbols/nuget/v3/index.json.
  /tmp/mono--msbuild-20221004-7860-1b2jkx2/Directory.Build.props(3,3): warning : Unable to load the service index for source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json.
  /tmp/mono--msbuild-20221004-7860-1b2jkx2/Directory.Build.props(3,3): warning : Unable to load the service index for source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json.
  /tmp/mono--msbuild-20221004-7860-1b2jkx2/Directory.Build.props(3,3): error : Unable to find package Microsoft.DotNet.Arcade.Sdk. No packages exist with this id in source(s): arcade, dotnet-public, dotnet-tools, dotnet3.1-symbols, dotnet5, dotnet5-transport
  /tmp/mono--msbuild-20221004-7860-1b2jkx2/Directory.Build.props(3,31): error MSB4236: The SDK 'Microsoft.DotNet.Arcade.Sdk' specified could not be found. [/tmp/mono--msbuild-20221004-7860-1b2jkx2/mono/build/sdks_and_nugets/sdks/sdks.csproj]
    Restoring package named Microsoft.NET.Sdk.Razor version='3.1.6'
  /tmp/mono--msbuild-20221004-7860-1b2jkx2/Directory.Build.props(3,31): error MSB4236: The SDK 'Microsoft.DotNet.Arcade.Sdk' specified could not be found. [/tmp/mono--msbuild-20221004-7860-1b2jkx2/mono/build/sdks_and_nugets/sdks/sdks.csproj]
```

Reply:
I think the last commit finally fixed this error!  Though now we have a different failure:
```
./stage1/mono-msbuild/msbuild: 1: Bad substitution
./stage1/mono-msbuild/msbuild: 1: Bad substitution
Cannot open assembly '/tmp/mono-20221004-13090-11sgbga/mono-6.12.0.182/MSBuild.dll': No such file or directory.
```
But nevertheless this is the most progress we've ever made in fixing `msbuild` on Linux.

Reply:
Probably need `sh --> bash` patch.

But not sure if changes were from `cert-sync` or `dotnet` so will need to do a few more runs dropping one or the other.

Reply:
So it was a certificate issue. I wonder if macOS build was able to use Keychain certs.

Now failing on `fsharp`. I think the build downloads a particular .NET SDK that needs ICU. Similar to `dotnet` formula issue.
```
Process terminated. Couldn't find a valid ICU package installed on the system. Set the configuration flag System.Globalization.Invariant to true if you want to run with no globalization support.
```